src/components/BotBuilder/BotWizard.js

```diff
@@ -95,12 +61,13 @@ class ContactBot extends React.Component {
 		return (
 			<div>
 				<StaticAppBar {...this.props} />
-				<div style={styles.home}>
+				<div style={styles.home} className="botbuilder-page-wrapper">
 					<Paper style={styles.paperStyle} className="botBuilder-page-card" zDepth={1}>
                         <Grid>
                             <Row>
-                                <Col xs={12} md={9}>
-                                    <div style={{width: '100%', maxWidth: 700, margin: 'auto'}}>
+                                <div style={{display: 'flex', 'flex-direction':window.innerWidth>769?'row':'column'}}>
```

Comment:
Can you try removing this line? I think the page is not responsive because of this line.

Reply:
This line is necessary, removing it shifts "preview" below the builder. I think `window.innerWidth>769` isn't working.